go.mod

```diff
@@ -122,6 +122,7 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/ory/dockertest/v3 v3.8.1
 	github.com/oschwald/geoip2-golang v1.8.0
+	github.com/percona/mongodb_exporter v0.31.2
```

Comment:
This is still a version of mongodb_exporter that has the licensing issues, while the version without it hasn't been released yet. 

This should be updated to SHA 6eedce7c9f79932592dad0496b49b2028992ff75 so we're importing the right thing

Reply:
you are totally right, fixing